### DISCUSSION TITLE
How to delete Downstream Consumers?

### DISCUSSION THREAD

**Discussion by yviph** [LCA-ActivityBrowser/activity-browser#973] [Q&A] [answered]
Hi all,
I wanted to create a multi-output process where I can modify the allocation factors manually.
While I was trying out things, I pulled one process to the other and now I have processes in the Downstream Consumers, which I do not want. Is there any option to delete the Downstream Consumers?

Thanks and kind regards, yviph
**
![grafik](https://github.com/LCA-ActivityBrowser/activity-browser/assets/134674186/bb346413-1787-450c-8fb1-1da9a4021530)
**

**Comment by marc-vdm** [comment, marked as answer]
Hi,

The table for downstream consumers is only to give you information on where your process is used. You can right click on these processes and open them, and remove this process as an input there. Hope this solves your problem!

**Reply by yviph** [reply]
Thank you!

**Reply by marc-vdm** [reply]
I'll keep this open so other people can also use this information easily :)

**Comment by OliverLyon** [comment]
Hi:

Does anyone know how to add product into downstream consumer section? I tried to drag from database but it did not work. Thanks.
